roles/cs.php-tideways/defaults/main.yml

```diff
@@ -0,0 +1,32 @@
+tideways_enable: yes
+tideways_environment: production
+tideways_api_key: ~
+tideways_auto_start: yes
+tideways_traces_only_keep_minimum_ms: 0
+
+# See: https://support.tideways.com/documentation/setup/configuration/configure-tideways-globally-via-php-ini.html
+tideways_service: web
+tideways_sample_rate: 25
+tideways_enable_cli: 0
+
+# If we're hosting magento optimize telling tideways which framework to expect
+tideways_framework: >-
```

Comment:
This ini setting is not required anymore with Tideways PHP version 5 and up (3 years old). You should not make this configurable and instead let the defaults work out here.